core/server/common/src/main/java/alluxio/master/audit/UserAccessAuditLog.java

```diff
@@ -0,0 +1,59 @@
+package alluxio.master.audit;
+
+import alluxio.RpcUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.ArrayBlockingQueue;
+
+public class UserAccessAuditLog {
+  private static final int QUEUE_SIZE = 100;
```

Comment:
This seems low. We don't ever want to hit this limit unless the rate of logging is lower than the rate of accesses. I suggest making it 10,000